graphql/src/adapters/api/fastapi.py

```diff
@@ -0,0 +1,33 @@
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+
+from src.adapters.api.graphql_app import graphql_app
+
+
+def create_fastapi_app():
+    app = FastAPI()
+
+    # TODO: Need to lock down origins, methods and headers
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=["*"],
+        allow_methods=["*"],
+        allow_headers=["*"],
+        allow_credentials=True,
+    )
+
+    # Create GraphQL routes
+    app.include_router(graphql_app, prefix="/graphql")
+
+    # Create health check
+    @app.get("/healthz")
+    def check_health():
```

Comment:
is this an alternative approach to create api routes? if yes, do we plan to use this instead of what we have in api-gateway/main.tf?

Reply:
Yup. ApiGateway will just have /graphql